tests/monitoring/vm_monitoring.go

```diff
@@ -304,3 +319,30 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 		})
 	})
 })
+
+func createAgentVMI() *v1.VirtualMachineInstance {
+	virtClient := kubevirt.Client()
+	agentVMI := libvmifact.NewFedora(libnet.WithMasqueradeNetworking()...)
+
+	By("Starting a VirtualMachineInstance")
+	agentVMI, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(agentVMI)).Create(context.Background(), agentVMI, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	libwait.WaitForSuccessfulVMIStart(agentVMI)
+
+	getOptions := metav1.GetOptions{}
+	var VMI *v1.VirtualMachineInstance
+
+	By("VMI has the guest agent connected condition")
+	Eventually(func() []v1.VirtualMachineInstanceCondition {
+		VMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(agentVMI)).Get(context.Background(), agentVMI.Name, getOptions)
+		Expect(err).ToNot(HaveOccurred())
+		return VMI.Status.Conditions
+	}, 240*time.Second, 1*time.Second).Should(
+		ContainElement(
+			MatchFields(
+				IgnoreExtras,
+				Fields{"Type": Equal(v1.VirtualMachineInstanceAgentConnected)})),
+		"Should have agent connected condition")
```

Comment:
this I think would become more readable if you could split to a variable with a meaningful name
`vmiAgentConnectedCondition`?

Reply:
I prefer to leave it as is. It is used only once.

Reply:
agreed it's not used more than once, and it's not related to performance, for other people and when we look back on this in a few weeks, having this variable would make it much easier to read and understand

Reply:
from what I could see in many other tests files, those optional descriptions are used like this, without a variable. I can't see why it's less readable.

Reply:
misunderstood earlier, done.